docs/docs/development/new-behavior.mdx

```diff
@@ -536,24 +855,50 @@ Zephyr currently does not support logging over Bluetooth, so any use of the seri
 Consider the following prompts when writing documentation for new behaviors:
 
 - What does it do? Describe some general use-cases for the behavior.
-- Which properties included in the [devicetree binding](#creating-the-devicetree-binding-yaml) should be configured manually by the user? What do they do, and if applicable, what are their default values?
+- Which properties included in the [devicetree binding](#devicetree-bindings-yaml) should be configured manually by the user? What do they do, and if applicable, what are their default values?
 - What does an example implementation in a keymap look like? Include a code-snippet of the example implementation in the keymap file's `behaviors` node.
 - How does the behavior perform in edge cases? For example, tap-dances invoke the last binding in its list of `bindings` once the maximum number of keypresses has been reached.
 
-Consider also including visual aids alongside written documentation if it adds clarity.
+Including visual aids alongside written documentation for additional clarity may be helpful.
+
+:::info
+If submitting a pull request, see [Documentation](contributing/documentation.md) for more information on writing, testing, and formatting ZMK documentation.
+:::
+
+## Licensing Information
+
+:::danger
+The ZMK Project and its contributors do not claim to be legal representatives, and any material below should not considered official legal advice.
+When distributing your work, please review the terms and conditions associated with any relevant licenses thoroughly.
+:::
+
+Developers may wish to share their work with the public, which is often done by sharing a link to a GitHub repository.
+However, making a repository public does **not** automatically qualify the repository as open source, or permit others to use the works as they see fit.
+Author must provide a license that in addition to their source code, satisfy criteria that includes but is not limited to:
```

Comment:
```suggestion
Authors should provide a license that in addition to their source code, ideally satisfies criteria that includes but is not limited to:
```
I softened this a bit, am I missing anything about the "must" clause?

Reply:
The "must" clause is specifically for qualifying the module as open source. 
https://opensource.org/osd
https://opensource.guide/legal/

Was trying to steer readers towards the open-source route given ZMK's disposition by setting up this sentence with the previous statement on public != open source